hash/m20.xml

```diff
@@ -14,6 +14,11 @@ license:CC0-1.0
   to properly access volume 1 ("VL 1: " gives error)
   OTOH, I have no idea how to boot the disks with sits on PCOS prompt...
 
+  EDIT: Added more floppies with games. Most of the original floppies in this file
+  have a missing first track. This is one reason for the error messages observed in the note above.
+  Track 0 does not contain any user data and can be transferred from a working floppy (e.g. pcos102).
+  More details: https://forums.bannister.org/ubbthreads.php?ubb=showflat&Number=100146#Post100146
```

Comment:
Don’t put revision history information here.  Please comment the individual disks with missing tracks and mark them as bad dumps.  Links to forum threads aren’t such a good idea.  If the missing tracks are due to FM not being readable in PC drives, say so.

Reply:
Will do, I'll need some time to go through the existing images to comment them.

Reply:
This should be solved.